.circleci/config.yml

```diff
@@ -137,8 +137,8 @@ jobs:
     steps:
       - setup_vm
       - run:
-          name: 'Install Protobuf'
-          command: pip3 install --user protobuf
+          name: 'Install Validator Dependencies'
```

Comment:
Let's try changing the `executor` on line 136 to `amphtml-xlarge-executor` and see how it affects CPP build time.

Reply:
After changes from 2 core medium executer to 8 core xlarge, the time of `amp validator-cpp` is reduced from [13m 11s](https://app.circleci.com/pipelines/github/ampproject/amphtml/5153/workflows/0577825e-61ab-411c-8b3d-17847b21b3ee/jobs/69433/parallel-runs/0/steps/0-109) to [3m 18s](https://app.circleci.com/pipelines/github/ampproject/amphtml/5304/workflows/3bf880c3-546c-411f-bac1-fa867be44e49/jobs/72100/parallel-runs/0/steps/0-109), and the total time of `Validator Tests` is reduced from ~19min to ~8min.

Reply:
That's excellent news! We're almost home.